### DISCUSSION TITLE
关于数据爬取类型价格筛选问题

### DISCUSSION THREAD

**Discussion by lingheaila** [mikumifa/BiliShareMall#3] [Q&A]
筛选价格似乎没生效
如图：
![image](https://github.com/user-attachments/assets/be080f14-1470-49f7-a5c4-46c18413285e)
结果：
![image](https://github.com/user-attachments/assets/324a1365-1359-4cb6-8228-b944ceece6ff)

日志：
[app.log](https://github.com/user-attachments/files/18201562/app.log)

**Comment by mikumifa** [comment]
b站的筛选也没办法选出价格里面的

**Reply by lingheaila** [reply]
![image](https://github.com/user-attachments/assets/e1bea3f4-e477-41de-9561-7367f684d0bf)
这个不是价格区间吗？
应该可以筛的吧，
就是这个：
![image](https://github.com/user-attachments/assets/a5340cc6-b089-47c2-8c85-b6d927753f71)
